README.md

```diff
@@ -251,9 +251,10 @@ Finds the latest blog post and updates the post date to the date specified. **Da
 
 ```bash
 npm run dev
+npm start
```

Comment:
@oktadeveloper/developer-advocates FYI... I changed the `npm run dev` command so you can pass in file names. As a result, it no longer runs `npm start`.

After this PR is merged, you can run the following if you just to NOT delete certain older posts:

```
npm run dev 2019-02-13-angular-schematics.adoc,2019-03-05-react-schematics.md,2019-05-21-vue-schematics.md
```

If you don't pass in any parameters, the behavior is the same as before (deletes all posts older than 2 weeks).

The previous behavior of restoring all blog posts when you run `git push` is [still intact](https://github.com/oktadeveloper/okta-blog/blob/main/package.json#L23).